src/com/oltpbenchmark/schema/TableSchema.java

```diff
@@ -0,0 +1,65 @@
+package com.oltpbenchmark.schema;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class TableSchema {
+    private final String name;
+    private final List<Column> columns;
+    private final String primaryKey;
+    private final String partitionKey;
+    
+    public TableSchema(String name, List<Column> columns, String primaryKey, String partitionKey) {
+        this.name = name;
+        this.columns = columns;
+        this.primaryKey = primaryKey;
+        this.partitionKey = partitionKey;
+    }
+    
+    public String name() {
+        return name;
+    }
+
+    public List<Column> getColumns() {
+        return columns;
+    }
+
+    public String getPrimaryKey() {
+        return primaryKey;
+    }
+
+    public String getPartitionKey() {
+        return partitionKey;
+    }
+}
+
+class TableSchemaBuilder {
+    private String name = null;
```

Comment:
1. `final`
2. `= null` is redundant
3. Initialize `name` in `TableSchemaBuilder` constructor as it is mandatory field

Reply:
Done